apio-consumer/src/main/kotlin/com/liferay/apio/consumer/graph/ApioGraph.kt

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.apio.consumer.graph
+
+import com.liferay.apio.consumer.model.Thing
+
+/**
+ * @author Javier Gamarra
+ */
+object ApioGraph {
```

Comment:
👏 👏 👏

Reply:
I would call this ApioCache or something like that, as it is not a graph :P

Reply:
Yep, it's not a graph xP, I just kept the same name from the old variable. ThingsCache is how you call it on the consumer-js. I'll use this one, right? @victorg1991

Reply:
Yes, I like that name too :)